conf/comitup.conf

```diff
@@ -6,10 +6,11 @@
 # ap_name
 #
 # This defines the name used for the AP hotspot, and for the ZeroConf
-# host name. The "<nnn>" string, if present, will be replaced with an
-# instance-unique, persistent number of the same length. There may be
-# up to four "n's" in the string. Similarly, the string "<hostname>"
-# is replaced with the system hostname.
+# host name. The "<nnn>|<MMMM>|<ssss>" string, if present, will be replaced
+# as follows,  <nnn> instance-unique, persistent number of the same length (up to 4)
+#              <MMMM> device WiFi MAC address of the same length (up to 12)
```

Comment:
Specify leading or trailing.
This also needs to be added to comitup-conf.5.md.

Reply:
I didn't realise leading was supported.

Reply:
<nnn> happens to be leading.